src/commands/ext-uninstall.ts

```diff
@@ -15,6 +18,26 @@ import { requirePermissions } from "../requirePermissions";
 import * as utils from "../utils";
 import * as logger from "../logger";
 
+marked.setOptions({
+  renderer: new TerminalRenderer(),
+});
+
+/**
+ * We do not currently support uninstalling extensions that require additional uninstall steps to be taken in the CLI. Direct them to the Console to uninstall the extension.
+ * - pubsub-stream-bigquery: requires calling a public HTTP uninstall function
+ *
+ * @param projectId id of the user's project
+ * @param instanceId id of the extension instance
+ * @return a void Promise
+ */
+function consoleUninstallOnly(projectId: string, instanceId: string): Promise<void> {
+  const instanceURL = `https://console.firebase.google.com/project/${projectId}/extensions/instances/${instanceId}`;
+  const consoleUninstall =
```

Comment:
Is there any reason to use Markdown here?  I assume the renderer will only underline the link, or does it somehow make it more clickable?

Elsewhere in the CLI codebase I'd expect:
```ts
import * as clc from "cli-color";
// ...
const consoleUninstall = `This extension can only be uninstalled through the Firebase Console. Please visit ${clc.bold(underline(instanceUrl))} to uninstall this extension`
```

Reply:
On certain terminals, this turns into a clickable link